### PR TITLE
Feature 13412 - Support for sbc-staff user

### PR DESCRIPTION
**Pull request by ritvick**
*Issue #:* /bcgov/entity/issues/13412

*Description of changes:*
Support for sbc-staff user

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -233,6 +233,7 @@ import {
   StaffPaymentOptions
 } from '@/enums'
 import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+import { consoleSandbox } from '@sentry/utils'
```
Comment (severinbeauvais) [inline src/App.vue:236]: This doesn't look like it's used. Please delete it.

File: src/App.vue [hunk 2]
```diff
@@ -1048,9 +1049,8 @@ export default class App extends Mixins(
 
   /** Gets authorizations from Auth API, verifies roles, and stores them. */
   private async checkAuth (id: string): Promise<any> {
-    // NB: will throw if API error
-    const authRoles = await AuthServices.fetchAuthorizations(id)
-    // NB: roles array may contain 'view', 'edit', 'staff' or nothing
+    const authRoles = getKeycloakRoles()
+    console.log('AuthRoles: ', authRoles)
```
Comment (severinbeauvais) [inline src/App.vue:1053]: This looks like a debugging statement. Please remove it.

**Review by severinbeauvais — COMMENTED** [review]
This looks ok but:
- please manually test the other code that uses `authRoles`
- please add a "Test Notes" comment in the ticket to inform QA what code paths need to be tested
- please fix the broken unit tests
- please update the app version

**Comment by seeker25** [comment]
I don't think this is necessary - there is no existing SBC-Staff keycloak role (STAFF exists but not SBC-STAFF). They're a single org - it makes it quite a bit easier when they can manage the org themselves

https://github.com/bcgov/sbc-auth/pull/2089

**Comment by ritvick** [comment]
New PR: https://github.com/bcgov/business-create-ui/pull/454

**Pull request closed without merging**